packages/cardano-services/src/Asset/TypeormAssetProvider/TypeormAssetProvider.ts

```diff
@@ -0,0 +1,44 @@
+import { Asset, AssetProvider, Cardano, GetAssetArgs } from '@cardano-sdk/core';
+import { NftMetadataService } from '../types';
+import { TypeormProvider, TypeormProviderDependencies } from '../../util';
+
+interface TypeormAssetProviderDependencies extends TypeormProviderDependencies {
+  nftMetadataService: NftMetadataService;
+}
+
+export class TypeormAssetProvider extends TypeormProvider implements AssetProvider {
```

Comment:
The card is not speaking about a new ` TypeormAssetProvider`, but sooner or later we will need it, so good to introduce it here.